core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalH2OBackend.scala

```diff
@@ -167,4 +171,23 @@ object InternalH2OBackend extends InternalBackendUtils {
       _.send(FlatFileMsg(nodes, conf.internalPortOffset))
     }
   }
+
+  private def initializeH2OHiveSupport(conf: H2OConf, user: String): Unit = {
+    if (conf.isHiveSupportEnabled) {
+      val tmpDir = Files.createTempDirectory("sparkling_water_hive_support").toFile
```

Comment:
Can we use `val sparkTmpDir = new File(Utils.getLocalDir(SparkEnv.get.conf))` defined in `org.apache.spark.expose.Utils` ? It will use spark tmp dir & we do not need to create our own. Spark also ensures it's deletion when it's shutting down

Reply:
It seems that the file is not needed anymore, thanks!